src/profilers.hpp

```diff
@@ -51,6 +52,7 @@ namespace aperf {
       void resume();
       void pause();
       int wait();
+      int handle_errors(int code, pid_t pid);
```

Comment:
If you decide to keep this method, it should be private because it's meant to be used only inside your class.

Reply:
Moved it inside process :)